ios/FluentUI/Vnext/AvatarGroup/AvatarGroup.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+
+/// `MSFAvatarGroupState` defines properties in an AvatarGroup
```

Comment:
for my own knowledge, is this the preferred commenting style of swiftui procotols? rather than definding these line above the actual var and functions?